components/src/core/Drawer/DrawerRailItem/DrawerRailItem.tsx

```diff
@@ -68,109 +82,137 @@ export type DrawerRailItemProps = SharedStyleProps & {
-        },
-        statusStripe: {
-            position: 'absolute',
-            top: 0,
-            bottom: 0,
-            left: 0,
-            width: 6,
-            zIndex: 100,
-            backgroundColor: (props): string => props.statusColor,
-        },
-        title: {
-            lineHeight: '1rem',
-            wordBreak: 'break-word',
-            hyphens: 'auto',
-            zIndex: 200,
-            color: (props): string => props.itemFontColor || theme.palette.text.primary,
+        [`& .${drawerRailItemClasses.condensed}`]: {
+            width: RAIL_WIDTH_CONDENSED,
+            minHeight: RAIL_WIDTH_CONDENSED,
         },
-        titleActive: {
-            fontWeight: 600,
+        [`& .${drawerRailItemClasses.ripple}`]: {
+            backgroundColor: theme.palette.primary.main,
         },
-    });
+    };
+});
+
+const ActiveItem = styled(Box, {
+    name: 'drawer-rail-item',
+    slot: 'active',
+})<Pick<DrawerRailItemProps, 'activeItemBackgroundColor'>>(({ activeItemBackgroundColor, theme }) => {
+    const fivePercentOpacityPrimary = color(
+        theme.palette.mode === 'dark' ? theme.palette.primary.dark : theme.palette.primary.main
+    )
+        .fade(0.95)
+        .string();
+    const twentyPercentOpacityPrimary = color(
+        theme.palette.mode === 'dark' ? theme.palette.primary.dark : theme.palette.primary.main
+    )
+        .fade(0.8)
+        .string();
+
+    return {
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        right: 0,
+        bottom: 0,
+        backgroundColor:
+            activeItemBackgroundColor ||
+            (theme.palette.mode === 'light' ? fivePercentOpacityPrimary : twentyPercentOpacityPrimary),
+    };
 });
 
+const StatusStripe = styled(Box, {
+    name: 'drawer-rail-item',
+    slot: 'statusStripe',
+})<Pick<DrawerRailItemProps, 'statusColor'>>(({ statusColor }) => ({
+    position: 'absolute',
+    top: 0,
+    bottom: 0,
+    left: 0,
+    width: 6,
+    zIndex: 100,
+    backgroundColor: statusColor,
+}));
+
+const Icon = styled(Avatar, {
+    name: 'drawer-rail-item',
+    slot: 'icon',
+})<Pick<DrawerRailItemProps, 'itemIconColor'>>(({ itemIconColor, theme }) => ({
+    color: itemIconColor || theme.palette.text.primary,
+    backgroundColor: 'transparent',
+    height: 'auto',
+    width: 'auto',
+    overflow: 'visible',
+}));
+
+const Title = styled(Typography, {
+    name: 'drawer-rail-item',
+    slot: 'title',
+})<Pick<DrawerRailItemProps, 'itemFontColor'>>(({ itemFontColor, theme }) => ({
+    lineHeight: '1rem',
+    wordBreak: 'break-word',
+    hyphens: 'auto',
+    zIndex: 200,
+    color: itemFontColor || theme.palette.text.primary,
+}));
+
+const DrawerRailItemDivider = styled(Divider, {
+    name: 'drawer-rail-item',
+    slot: 'divider',
+})(() => ({
+    position: 'absolute',
+    bottom: 0,
+    left: 0,
+    width: '100%',
+    [`& .${drawerRailItemClasses.itemActive}`]: {
```

Comment:
Did you change this class name from `titleActive` to `itemActive` ?

Reply:
No. There are 2 separate classes titleActive and itemActive.